tests/network-tests/src/Api.ts

```diff
@@ -859,6 +862,42 @@ export class Api {
     ])
   }
 
+  // Membership
+  async updateMemberControllerAccount(
+    oldAccount: string,
+    memberId: MemberId,
+    newAccount: string
+  ): Promise<ISubmittableResult> {
+    const updateControllerAccount = this.api.tx.members.updateAccounts(memberId, newAccount, newAccount)
+    await this.prepareAccountsForFeeExpenses(oldAccount, [updateControllerAccount])
+    return this.sender.signAndSend(updateControllerAccount, oldAccount)
+  }
+
+  async updateCouncillorsAccounts(initialBalance = KNOWN_COUNCILLOR_ACCOUNT_DEFAULT_BALANCE): Promise<string[]> {
+    const memberIds = (await this.query.council.councilMembers()).map((m) => m.membershipId)
+    const memberRootAccounts = await Promise.all(
+      memberIds.map(async (id) => {
+        const membership = await this.query.members.membershipById(id)
+        return membership.unwrap().rootAccount.toString()
+      })
+    )
+    // path to append to base SURI
+    const debug = extendDebug('api-factory')
+    debug(`assigning Well Known Councillors Account`)
+    const newAccounts = memberIds.map((id) => {
+      const uri = `//Councillor//` + id.toString()
```

Comment:
I noticed there is a small issue here. Because the `uri` starts with `//` and `isFinalPath = false`, the final uri becomes:
```typescript
`${miniSecret}//testing////Councillor//${memberId}`
```
_(notice 4 slashes before `Councillor`)_